src/components/Explore/TokenRow.tsx

```diff
@@ -183,13 +195,19 @@ const PercentChangeInfoCell = styled(Cell)`
     line-height: 16px;
   }
 `
-const PriceCell = styled(Cell)`
+const PriceCell = styled(Cell)<{ sortable: boolean }>`
   justify-content: flex-end;
   min-width: 80px;
+  padding-right: 4px;
 
   @media only screen and (max-width: ${MOBILE_MEDIA_BREAKPOINT}) {
     min-width: max-content;
   }
+
+  &:hover {
+    color: ${({ theme, sortable }) => sortable && theme.text1};
+    background-color: ${({ theme, sortable }) => sortable && darken(0.08, theme.bg0)};
```

Comment:
i see darken a lot - wonder if we should ask the design team whether to create a new color for our theme

Reply:
Yup! one of our tickets is the design theme file for Explore